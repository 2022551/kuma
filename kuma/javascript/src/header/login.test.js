import React from 'react';
import { create } from 'react-test-renderer';
import Login from './login.jsx';

test('Login snapshot', () => {
    const login = create(<Login />);
    expect(login.toJSON()).toMatchSnapshot();
});
