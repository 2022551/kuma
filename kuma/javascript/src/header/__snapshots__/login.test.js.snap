// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login snapshot 1`] = `
.emotion-1 {
  font-size: 18px;
  font-weight: bold;
  color: black;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.emotion-1:hover {
  -webkit-text-decoration: none;
  text-decoration: none;
}

.emotion-0 {
  vertical-align: -6px;
}

<a
  className="emotion-1"
  data-service="GitHub"
  href="/users/github/login/?next=/"
  rel="nofollow"
>
  Sign in
   
  <svg mock
    className="emotion-0"
  />
</a>
`;
